conda-recipe/meta.yaml

```diff
@@ -52,7 +52,11 @@ requirements:
         - sphinxcontrib-bibtex
         - docutils =0.16 # This dependency is not explicity needed but https://github.com/NNPDF/nnpdf/issues/1220
         - curio >=1.0
-        - pineappl >=0.5.2
+        - pineappl >=0.5.8
+        - eko
+        - banana-hep
+        - lz4 >=3.1.10
```

Comment:
Would be good to have a comment on this.

Reply:
```suggestion
        - lz4 >=3.1.10 # see https://github.com/conda-forge/eko-feedstock/issues/10
```